spec/features/decision_spec.rb

```diff
@@ -0,0 +1,23 @@
+feature "Decision" do
+  scenario "Player chooses rock button" do
+    visit ('/names')
+    fill_in :player_name, with: "Ana"
+    click_button "Start game"
+    click_button("Rock")
+    expect(page).to have_content "You have picked rock"
```

Comment:
Love the thorough tests and that your pages shows the users choice before declaring a winner